frontend/insurance/src/app/modules/edit-claim/edit-vehicle/edit-vehicle.component.html

```diff
@@ -7,59 +7,113 @@ <h2 lyTyp="headline" gutterBottom>
-              #_input
-              lyNativeControl
-              formControlName="model"
-              type="text"
-            />
-            <span style="color:orangered">Required</span>
-          </ly-field>
+            Model
+          </h1>
+          <h1 lyTyp="headline" *ngIf="!isEditing" class="center-text">
+            {{ claim.vehicle.model }}
+          </h1>
+          <div class="auto-margin center-text width-75">
+            <ly-field
+              *ngIf="isEditing"
+              fullWidth
+              [appearance]="appearance.value"
+              class="mts mbn"
+            >
+              <input
+                #_input
+                lyNativeControl
+                formControlName="model"
+                type="text"
+                class="center-text"
+              />
+              <ly-hint
+                ><span *ngIf="isEditing" style="color:orangered;"
+                  >Required</span
+                ></ly-hint
+              >
+            </ly-field>
+          </div>
         </ly-grid>
         <hr *ngIf="!isEditing" />
         <ly-grid item col="12">
-          <h1 lyTyp="display1" color="offPrimary">Year</h1>
-          <h1 lyTyp="headline" *ngIf="!isEditing">{{ claim.vehicle.year }}</h1>
-          <ly-field
-            *ngIf="isEditing"
-            fullWidth
-            [appearance]="appearance.value"
-            class="mtn"
+          <h1
+            lyTyp="display1"
+            color="offPrimary"
+            class="center-text edit-claim-border width-75 auto-margin"
           >
-            <input #_input lyNativeControl formControlName="year" type="text" />
-            <span style="color:orangered">Required</span>
-          </ly-field>
+            Year
+          </h1>
+          <h1 lyTyp="headline" *ngIf="!isEditing" class="center-text">
+            {{ claim.vehicle.year }}
+          </h1>
+          <div class="auto-margin center-text width-75">
+            <ly-field
+              *ngIf="isEditing"
+              fullWidth
+              [appearance]="appearance.value"
+              class="mts mbn"
+            >
+              <input
+                #_input
+                lyNativeControl
+                formControlName="year"
+                type="text"
+                class="center-text"
+              />
+              <ly-hint
+                ><span *ngIf="isEditing" style="color:orangered;"
+                  >Required</span
+                ></ly-hint
+              >
+            </ly-field>
+          </div>
         </ly-grid>
         <hr *ngIf="!isEditing" />
         <ly-grid item col="12">
-          <h1 lyTyp="display1" color="offPrimary">Color</h1>
-          <h1 lyTyp="headline" *ngIf="!isEditing">{{ claim.vehicle.color }}</h1>
-          <ly-field
-            *ngIf="isEditing"
-            fullWidth
-            [appearance]="appearance.value"
-            class="mtn"
+          <h1
+            lyTyp="display1"
+            color="offPrimary"
```

Comment:
The reason I made offPrimary was so that we can have headers (Like the big "Edit Claim" header at the top of this page) show easily. These headers can be shown in just regular 'primary'